.github/CONTRIBUTING.md

```diff
@@ -1,388 +1,98 @@
 # Contributing
 
-## Requirements
+## Prerequisites
 
-Set up your SSH Key GitHub account and install node.js 4 or higher.
+To contribute to Carbon, you'll need to install:
 
-- [Generating SSH Keys - GitHub](https://help.github.com/articles/generating-ssh-keys/)
-- [`nvm` (Node Version Manager)](https://github.com/creationix/nvm) to use the
-  `Node 6`.
+- [node(v10 or higher)](https://nodejs.org/en/download/)(you can download it
+  here or follow their installation through a package manager
+  [here](https://nodejs.org/en/download/package-manager/))
+  - We recommend using
+    [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md) as your version
+    manager for Node.
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [yarn](https://yarnpkg.com/en/docs/install#mac-stable)
 
-Contributing to carbon-components requires that you can run this repo locally on
-your computer.
+You will also need to have a GitHub account. Head over to
```

Comment:
Do we need this section? Not sure what we're assuming when someone heads to this file, we might be able to assume they have a GitHub account but I'm not sure

Reply:
That's fair! I was mainly following what was previously there but considering this is stored in Github, it's likely they have an account.

Reply:
Not sure if the couple sentences here on GitHub and Yarn help: https://www.carbondesignsystem.com/tutorial/react/overview#yarn